SimG4CMS/Forward/src/TotemRPOrganization.cc

```diff
@@ -35,15 +35,15 @@ uint32_t TotemRPOrganization ::getUnitID(const G4Step* aStep) const {
   G4VPhysicalVolume* physVol;
   int32_t UNITA = 0;
   const G4VTouchable* touch = aStep->GetPreStepPoint()->GetTouchable();
-  int ii = 0;
-  for (ii = 0; ii < touch->GetHistoryDepth(); ii++) {
-    physVol = touch->GetVolume(ii);
 
 #ifdef SCRIVI
+  for (int ii = 0; ii < touch->GetHistoryDepth(); ii++) {
+    physVol = touch->GetVolume(ii);
```

Comment:
Hopefully this call has no other purpose than returning the physics volume: correct @civanch ?

Reply:
@perrotta , yes, it is.